### ISSUE TITLE
environment.yaml

### ISSUE THREAD

**Issue by GalleonsL** [SenseTime-Knowledge-Mining/BridgeDPI#2]
The `environment.yaml` file is missing from the project.

**Comment by GalleonsL** [comment]
name: bridgedpi
channels:
  - pytorch
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
  - conda-forge
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/pytorch/
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main/
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free
  - defaults
dependencies:
  - _libgcc_mutex=0.1=conda_forge
  - _openmp_mutex=4.5=2_kmp_llvm
  - _tflow_select=2.3.0=mkl
  - abseil-cpp=20211102.0=hd4dd3e8_0
  - absl-py=1.3.0=py38h06a4308_0
  - aiohttp=3.8.3=py38h5eee18b_0
  - aiosignal=1.2.0=pyhd3eb1b0_0
  - astor=0.8.1=py38h06a4308_0
  - asttokens=2.2.1=pyhd8ed1ab_0
  - astunparse=1.6.3=py_0
  - async-timeout=4.0.2=py38h06a4308_0
  - attrs=22.1.0=py38h06a4308_0
  - backcall=0.2.0=pyh9f0ad1d_0
  - backports=1.0=pyhd8ed1ab_3
  - backports.functools_lru_cache=1.6.4=pyhd8ed1ab_0
  - blas=1.0=mkl
  - blinker=1.4=py38h06a4308_0
  - boost=1.74.0=py38h2b96118_5
  - boost-cpp=1.74.0=h75c5d50_8
  - bottleneck=1.3.5=py38h7deecbd_0
  - brotlipy=0.7.0=py38h27cfd23_1003
  - bzip2=1.0.8=h7f98852_4
  - c-ares=1.19.0=h5eee18b_0
  - ca-certificates=2023.01.10=h06a4308_0
  - cachetools=4.2.2=pyhd3eb1b0_0
  - cairo=1.16.0=hb05425b_3
  - certifi=2022.12.7=py38h06a4308_0
  - cffi=1.15.1=py38h5eee18b_3
  - charset-normalizer=2.0.4=pyhd3eb1b0_0
  - click=8.0.4=py38h06a4308_0
  - cryptography=39.0.1=py38h9ce1e76_0
  - cudatoolkit=10.2.89=hfd86e86_1
  - cycler=0.11.0=pyhd8ed1ab_0
  - debugpy=1.5.1=py38h295c915_0
  - decorator=5.1.1=pyhd8ed1ab_0
  - deepchem=2.7.1=pyhd8ed1ab_0
  - entrypoints=0.4=pyhd8ed1ab_0
  - executing=1.2.0=pyhd8ed1ab_0
  - expat=2.5.0=hcb278e6_1
  - fontconfig=2.14.2=h14ed4e7_0
  - freetype=2.12.1=h4a9f257_0
  - frozenlist=1.3.3=py38h5eee18b_0
  - gast=0.4.0=pyhd3eb1b0_0
  - gensim=4.3.0=py38h6a678d5_0
  - giflib=5.2.1=h5eee18b_3
  - glib=2.69.1=he621ea3_2
  - google-auth=2.6.0=pyhd3eb1b0_0
  - google-auth-oauthlib=0.4.4=pyhd3eb1b0_0
  - google-pasta=0.2.0=pyhd3eb1b0_0
  - greenlet=2.0.1=py38h6a678d5_0
  - grpc-cpp=1.48.2=h5bf31a4_0
  - grpcio=1.48.2=py38h5bf31a4_0
  - h5py=2.10.0=py38hd6299e0_1
  - hdf5=1.10.6=h3ffc7dd_1
  - icu=70.1=h27087fc_0
  - idna=3.4=py38h06a4308_0
  - importlib-metadata=6.0.0=py38h06a4308_0
  - intel-openmp=2021.4.0=h06a4308_3561
  - ipykernel=6.15.0=pyh210e3f2_0
  - ipython=8.12.0=pyh41d4057_0
  - jedi=0.18.2=pyhd8ed1ab_0
  - joblib=1.2.0=pyhd8ed1ab_0
  - jpeg=9e=h5eee18b_1
  - jupyter_client=7.0.6=pyhd8ed1ab_0
  - jupyter_core=5.3.0=py38h578d9bd_0
  - keras-preprocessing=1.1.2=pyhd3eb1b0_0
  - kiwisolver=1.4.4=py38h6a678d5_0
  - lcms2=2.12=h3be6417_0
  - ld_impl_linux-64=2.38=h1181459_1
  - lerc=3.0=h295c915_0
  - libdeflate=1.17=h5eee18b_0
  - libexpat=2.5.0=hcb278e6_1
  - libffi=3.4.2=h6a678d5_6
  - libgcc-ng=12.2.0=h65d4601_19
  - libgfortran-ng=12.2.0=h69a702a_19
  - libgfortran5=12.2.0=h337968e_19
  - libpng=1.6.39=h5eee18b_0
  - libprotobuf=3.20.3=he621ea3_0
  - libsodium=1.0.18=h36c2ea0_1
  - libstdcxx-ng=11.2.0=h1234567_1
  - libtiff=4.5.0=h6a678d5_2
  - libuuid=2.38.1=h0b41bf4_0
  - libwebp=1.2.4=h11a3e52_1
  - libwebp-base=1.2.4=h5eee18b_1
  - libxcb=1.15=h7f8727e_0
  - libzlib=1.2.13=h166bdaf_4
  - llvm-openmp=16.0.1=h417c0b6_0
  - lz4-c=1.9.4=h6a678d5_0
  - markdown=3.4.1=py38h06a4308_0
  - markupsafe=2.1.1=py38h7f8727e_0
  - matplotlib-base=3.4.3=py38hf4fb855_1
  - matplotlib-inline=0.1.6=pyhd8ed1ab_0
  - mkl=2021.4.0=h06a4308_640
  - mkl-service=2.4.0=py38h7f8727e_0
  - mkl_fft=1.3.1=py38hd3c417c_0
  - mkl_random=1.2.2=py38h51133e4_0
  - multidict=6.0.2=py38h5eee18b_0
  - ncurses=6.4=h6a678d5_0
  - nest-asyncio=1.5.6=pyhd8ed1ab_0
  - ninja=1.10.2=h06a4308_5
  - ninja-base=1.10.2=hd09550d_5
  - numexpr=2.8.4=py38he184ba9_0
  - numpy=1.23.5=py38h14f4228_0
  - numpy-base=1.23.5=py38h31eccc5_0
  - oauthlib=3.2.2=py38h06a4308_0
  - openssl=1.1.1t=h7f8727e_0
  - opt_einsum=3.3.0=pyhd3eb1b0_1
  - packaging=23.1=pyhd8ed1ab_0
  - pandas=1.5.3=py38h417a72b_0
  - parso=0.8.3=pyhd8ed1ab_0
  - pcre=8.45=h9c3ff4c_0
  - pexpect=4.8.0=pyh1a96a4e_2
  - pickleshare=0.7.5=py_1003
  - pillow=9.4.0=py38h6a678d5_0
  - pip=23.0.1=py38h06a4308_0
  - pixman=0.40.0=h36c2ea0_0
  - platformdirs=3.2.0=pyhd8ed1ab_0
  - prompt-toolkit=3.0.38=pyha770c72_0
  - prompt_toolkit=3.0.38=hd8ed1ab_0
  - protobuf=3.20.3=py38h6a678d5_0
  - psutil=5.9.5=py38h1de0b5d_0
  - ptyprocess=0.7.0=pyhd3deb0d_0
  - pure_eval=0.2.2=pyhd8ed1ab_0
  - pyasn1=0.4.8=pyhd3eb1b0_0
  - pyasn1-modules=0.2.8=py_0
  - pycairo=1.23.0=py38h190342e_0
  - pycparser=2.21=pyhd3eb1b0_0
  - pygments=2.15.1=pyhd8ed1ab_0
  - pyjwt=2.4.0=py38h06a4308_0
  - pyopenssl=23.0.0=py38h06a4308_0
  - pyparsing=3.0.9=pyhd8ed1ab_0
  - pysocks=1.7.1=py38h06a4308_0
  - python=3.8.16=h7a1cb2a_3
  - python-dateutil=2.8.2=pyhd8ed1ab_0
  - python-flatbuffers=2.0=pyhd3eb1b0_0
  - python_abi=3.8=2_cp38
  - pytorch=1.6.0=py3.8_cuda10.2.89_cudnn7.6.5_0
  - pytz=2023.3=pyhd8ed1ab_0
  - pyzmq=19.0.2=py38ha71036d_2
  - rdkit=2022.03.2=py38ha829ea6_0
  - re2=2022.04.01=h295c915_0
  - readline=8.2=h5eee18b_0
  - reportlab=3.6.12=py38h8786f81_2
  - requests=2.28.1=py38h06a4308_1
  - requests-oauthlib=1.3.0=py_0
  - rsa=4.7.2=pyhd3eb1b0_1
  - scikit-learn=1.2.2=py38h6a678d5_0
  - scipy=1.9.3=py38h14f4228_1
  - setuptools=66.0.0=py38h06a4308_0
  - six=1.16.0=pyhd3eb1b0_1
  - smart_open=6.3.0=pyhd8ed1ab_1
  - sqlalchemy=2.0.9=py38h1de0b5d_0
  - sqlite=3.41.2=h5eee18b_0
  - stack_data=0.6.2=pyhd8ed1ab_0
  - tensorboard=2.11.0=py38h06a4308_0
  - tensorboard-data-server=0.6.1=py38h52d8a92_0
  - tensorboard-plugin-wit=1.8.1=py38h06a4308_0
  - tensorflow=2.4.1=mkl_py38hb2083e0_0
  - tensorflow-base=2.4.1=mkl_py38h43e0292_0
  - tensorflow-estimator=2.6.0=pyh7b7c402_0
  - termcolor=2.1.0=py38h06a4308_0
  - threadpoolctl=3.1.0=pyh8a188c0_0
  - tk=8.6.12=h1ccaba5_0
  - torchvision=0.7.0=py38_cu102
  - tornado=6.3=py38h1de0b5d_0
  - tqdm=4.65.0=py38hb070fc8_0
  - traitlets=5.9.0=pyhd8ed1ab_0
  - typing-extensions=4.5.0=hd8ed1ab_0
  - typing_extensions=4.5.0=pyha770c72_0
  - urllib3=1.26.15=py38h06a4308_0
  - wcwidth=0.2.6=pyhd8ed1ab_0
  - werkzeug=2.2.3=py38h06a4308_0
  - wheel=0.38.4=py38h06a4308_0
  - wrapt=1.14.1=py38h5eee18b_0
  - xz=5.2.10=h5eee18b_1
  - yarl=1.8.1=py38h5eee18b_0
  - zeromq=4.3.4=h9c3ff4c_1
  - zipp=3.11.0=py38h06a4308_0
  - zlib=1.2.13=h166bdaf_4
  - zstd=1.5.5=hc292b87_0
prefix: /home/galleons/anaconda3/envs/bridgedpi
